### ISSUE TITLE
TorrentDay ignoring apostrophes in search

### ISSUE THREAD

**Issue by 13rennon** [Jackett/Jackett#15255]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
Using HTTP Client: HttpWebClient2
Using FlareSolverr: No
Using proxy: Disabled
App config/log directory: C:\ProgramData\Jackett
ThreadPool MaxThreads: 1023 workerThreads, 1000 completionPortThreads
Running in Docker: No
Jackett variant: CoreWindows
OS version: Microsoft Windows NT 10.0.22631.0 (64bit OS)
Environment version: 6.0.29 (C:\ProgramData\Jackett\)
Starting Jackett v0.21.2395
```

### Description

Using TorrentDay indexer via Jackett with Sonarr 4.0.4.1491. For whatever reason, Sonarr is sending the search request with special characters but Jackett/TorrentDay seem to be ignoring them? 

For example, Sonarr log shows:
`2024-04-17 14:24:09.1|Info|ReleaseSearchService|Searching indexers for [Mike Judge's Beavis and Butt-Head : S02E15]. 1 active indexers
2024-04-17 14:24:09.4|Info|DownloadDecisionMaker|No results found`

Jackett log shows: 
`2024-04-17 15:04:38 | Info | Torznab search in TorrentDay for Mike Judges Beavis and Butt Head S02E15 => Found 0 releases [238ms]`

The problem is that when I manually search on the TD website or with the manual search in Jackett I only get results if the apostrophe is included so Sonarr isn't giving me any results for select series/torrents. Is there any way to include an option for negating special cahracters?

### Logged Error Messages

```text
2024-04-17 15:04:38 | Info | Torznab search in TorrentDay for Mike Judges Beavis and Butt Head S02E15 => Found 0 releases [238ms]
```

**Comment by garfield69** [comment]
Yes, I believe we can enable SupportsRawSearch = true on the torznab.caps section of the C# TorrentDay indexer.
Unfortunately none of the current team have an account on TD so we cannot test independently.
I'll raise a PR and you can DL the windows build and see if it works. I'll update once I've the PR done.

**Comment by 13rennon** [comment]
I have a few invites for TD if you'd like one, just pm me your email.

**Comment by garfield69** [comment]
I used to have an account but it got deleted, so there is little point getting another one as I obviously don't do enough activity/downloads to retain an account.

Please send an invite to `jacketttest [at] gmail [dot] com` and another team member can do the testing ;-) Thanks.

**Comment by 13rennon** [comment]
Done

**Comment by ilike2burnthing** [comment]
Thanks for the invite.

I've been working on this for a few hours, but can't get it to work. The issue isn't limited to TorrentDay however, as other indexers with raw search enabled also just search for `Mike Judges` rather than `Mike Judge's`.

I previously saw something similar with 90 Day Fiancé, but never found out the cause - https://forums.sonarr.tv/t/sonarr-searching-for-different-keywords-across-indexers/32981 (similar to my suggestion there, this would need to be discussed on their Discord).

That aside though, I'm not sure TorrentDay needs raw search. While there are some results which use `Mike Judge's`, the majority do not. Results for Grey's Anatomy and Zoey's Extraordinary Playlist don't use apostrophes either, and diacritics appear to be ignored - searching using either `la` or `lá` will return the same number of results, with both `la` and `lá`.

If there's something which would benefit from raw search that I've missed, let me know.
